webservices/resources/reports.py

```diff
@@ -46,25 +50,35 @@ def parse_types(types):
             'type': 'string',
         },
         {
+
             'name': 'committee_type',
             'in': 'path',
             'type': 'string',
-            'description': 'House, Senate or presidential',
-            'enum': ['presidential', 'pac-party', 'house-senate'],
+            'description': 'House, Senate, presidential, independent expenditure only',
+            'enum': ['presidential', 'pac-party', 'house-senate', 'ie-only'],
         },
     ],
 )
 class ReportsView(Resource):
 
+    @schemas.marshal_with(schemas.CommitteeReportsPageSchema(), wrap=False)
```

Comment:
To document the conversation we had- we decided it would be best to document have all the reports share a schema and an endpoint. 

We will improve the documentation by displaying all the fields in swagger, instead of just the shared reports fields. (I think that can be a different PR.)

Also, I am going to add what form the information is coming from,  to make things slightly more clear.